pandas/core/arrays/numeric.py

```diff
@@ -130,3 +158,16 @@ def _arith_method(self, other, op):
             )
 
         return self._maybe_mask_result(result, mask, other, op_name)
+
+    def _apply(self, func: Callable, **kwargs) -> "NumericArray":
+        values = self._data[~self._mask]
+        values = np.round(values, **kwargs)
+
+        data = np.zeros(self._data.shape)
+        data[~self._mask] = values
+        return type(self)(data, self._mask)
```

Comment:
The mask needs to be copied I think? (result should not share a mask with the original array, because otherwise editing one can modify the other. We should probably also test this)